tensorboard/examples/plugins/example_basic/README.md

```diff
@@ -2,51 +2,160 @@
 
 ## Overview
 
-You know (and, we hope, love!) TensorBoard’s core features like the scalar dashboard, the graph explorer, and the embedding projector. However, in every TensorBoard user’s life, there comes a time when they want some cool new visualization that just…doesn’t exist yet. That’s what the TensorBoard plugin system is for.
+You know (and, we hope, love!) TensorBoard’s core features! However, in every TensorBoard user’s life, there comes a time when you want some cool new visualization that just…doesn’t exist yet. That’s what the TensorBoard plugin system is for.
```

Comment:
> just…doesn’t 

`just… doesn't?` Removing ellipsis might be even better.

Reply:
Done